python/composio/client/collections.py

```diff
@@ -386,6 +386,7 @@ class TriggerConfigPropertyModel(BaseModel):
 
     description: str
     title: str
+    default: t.Any
```

Comment:
Consider adding type validation for the `default` field based on the field's `type` property. This would help catch type mismatches between default values and expected field types early.